docs/METRICS.md

```diff
@@ -498,6 +498,8 @@ This sampler provides telemetry about TCP traffic and connections.
 * `tcp/connection/accepted` - number of connections accepted passively
 * `tcp/connection/initiated` - number of connections initiated actively
 * `tcp/drop` - number of packets dropped in the kernel TCP stack
+* `tcp/tlp` - number of Tail Loss Recovery Probes sent
+* `tcp/retransmit_timeout` - number of retransmit timeouts
```

Comment:
Should this be `tcp/transmit/retransmit_timeout` ?

Reply:
Yes, it missed that. I changed it now to group it.